eng/deploy.yaml

```diff
@@ -0,0 +1,148 @@
+parameters:
+  Subscription: ''
+  ServiceFabricConnection: ''
+  PublishProfile: ''
+  DotNetStatusAppName: ''
+  MaestroUrl: ''
+variables:
+  Subscription: ${{ parameters.Subscription }}
+  ServiceFabricConnection: ${{ parameters.ServiceFabricConnection }}
+  PublishProfile: ${{ parameters.PublishProfile }}
+  DotNetStatusAppName: ${{ parameters.DotNetStatusAppName }}
+  MaestroUrl: ${{ parameters.MaestroUrl }}
+
+  # --- Secret Variable group requirements ---
+  # maestro-admin-sql-connection-string
+  # maestro-token
+  # dn-bot-dnceng-build-rw-code-rw-release-rw
+  # maestro-scenario-test-github-token
+
+stages:
+- stage: deploy
+  displayName: Deploy
+  dependsOn: build
+  jobs:
```

Comment:
Prod deployment needs a approval job before any deployment happens.

Reply:
Is it this:
```
    - deployment: Approval
      displayName: approval for deployment to prod environment
      pool:
        name: Hosted VS2017
      environment: 'Production'
      strategy:
        runOnce:
          deploy:
            steps:
            - download: none
```

---
In reply to: [344314633](https://github.com/dotnet/arcade-services/pull/756#discussion_r344314633) [](ancestors = 344314633)

Reply:
I... think I did this?

---
In reply to: [344357832](https://github.com/dotnet/arcade-services/pull/756#discussion_r344357832) [](ancestors = 344357832,344314633)

Reply:
Yes that's how you add approval. The environments need to be manually created in the account, in this case dnceng.